sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -43,24 +44,23 @@ def _return_response(pipeline_response, deserialized, response_headers):
 
 
 class ContainerRegistryClient(ContainerRegistryBaseClient):
-    def __init__(self, endpoint, credential=None, **kwargs):
-        # type: (str, Optional[TokenCredential], **Any) -> None
+    def __init__(self, endpoint, credential=None, audience=AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD, **kwargs):
+        # type: (str, Optional[TokenCredential], Optional[str], **Any) -> None
```

Comment:
This parameter should remain keyword-only.

Reply:
Also, the type is not `Optional[str]` because `None` is not a valid value. Just `str`.

Reply:
While we're here - could we also update the constructor method signature to use Python3 syntax?